cli/command/formatter/stats.go

```diff
@@ -37,11 +36,72 @@ type ContainerStatsAttrs struct {
 	PidsCurrent      uint64 // Not used on Windows
 }
 
-// ContainerStats represents the containers statistics data.
+// ContainerStatsRecord represents the statistics data collected from a container.
```

Comment:
Still not quite sure what this does. Maybe it's just for formatting? 
Maybe `ContainerStatsFormatter`?